packages/components/src/ErrorView.scss

```
@@ -0,0 +1,83 @@
+@import '../scss/custom.scss';
+
+.error-view {
+  position: relative;
+  color: $danger;
+  border-radius: $border-radius;
+  background-color: negative-opacity($exception-transparency);
+  display: flex;
+  flex-direction: column;
+  flex-grow: 0;
+  font-family: $font-family-monospace;
+  transition: all $transition ease-in-out;
+  max-height: 150px;
+
+  &.expanded {
+    max-height: 100%;
+  }
+
+  .error-view-header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    text-wrap: nowrap;
+    width: 100%;
+
+    .error-view-header-text {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      padding-left: $spacer;
+      padding-right: $spacer;
+      font-weight: bold;
+      flex-shrink: 1;
+      overflow: hidden;
+      white-space: nowrap;
+
+      span {
+        flex-shrink: 1;
+        overflow: hidden;
+        white-space: nowrap;
+        text-overflow: ellipsis;
+        padding-left: $spacer-1;
+      }
+    }
+
+    .error-view-buttons {
+      display: flex;
+      flex-direction: row;
+      gap: 1px;
+      overflow: hidden;
+      border-radius: 0 $border-radius 0 0;
+      flex-shrink: 0;
+
+      .btn-danger {
+        border-radius: 0;
+        color: $black;
+        opacity: 0.8;
+        padding: $spacer-1;
+        &:active {
+          color: $black;
```

```suggestion
        color: var(--dh-color-contrast-dark);
        opacity: 0.8;
        padding: $spacer-1;
        &:active {
          color: var(--dh-color-contrast-dark);
```

Got to watch those color variables for light theme.

---

We could try adding a stylelint rule to ban sass variables basically. I think we could use [this rule](https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/dollar-variable-pattern/README.md) and set the regex to something like `/iWillNotUseSassVariables/` which would effectively ban sass variables I think

---

Would need rules to allow for $spacer as well.